core-bundle/src/Mailer/ContaoMailer.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Mailer;
+
+use Contao\PageModel;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\Mailer\Envelope;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
+use Symfony\Component\Mime\Message;
+use Symfony\Component\Mime\RawMessage;
+
+final class ContaoMailer implements MailerInterface
```

Comment:
Is it necessary to make this class final? Because https://github.com/contao/contao/issues/924.

Reply:
It is an implementation of the `MailerInterface` and never needs to be mocked. Imo final is correct here.